gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/SignModule.kt

```diff
@@ -314,16 +324,7 @@ open class SignModule @Inject constructor(_providers: ProviderFactory, _objects:
 
         logger.debug("Signed module will be named ${signed.get().asFile.absolutePath}")
 
-        signModule(
-            keystore.getOrNull(),
-            pkcs11Cfg.getOrNull(),
-            keystorePw.get(),
-            certFile.get(),
-            certPw.get(),
-            alias.get(),
-            unsignedModule,
-            signed.get().asFile
-        )
+        signModule(unsignedModule)
```

Comment:
Sonar was whining about the number of args. Plus all but `unsignedModule` are instance properties.